static/js/index.js

```diff
@@ -0,0 +1,127 @@
+const API_PREFIX = 'api';
+const baseUrl = window.location.origin;
+
+const setScrapingLogs = (logs) => {
+  const logsNode = document.getElementById('scraping-logs');
+  const root = document.createElement('div');
+  logs.forEach((el) => {
+    const pNode = document.createElement('p');
+    pNode.innerText = el;
```

Comment:
Try to use template literals

Reply:
I think I can't use template literals in this case due to the for-loop, I would use innerHTML but it overwrites the previous content, so I need to use appendChild